README.md

```diff
@@ -20,18 +22,28 @@ See details at [wiki](https://github.com/miripiruni/CSScomb/wiki/Requirements).
 
 ```
 cd ~/.vim/bundle
-git clone https://github.com/miripiruni/CSScomb-for-Vim.git
+git clone https://github.com/csscomb/vim-csscomb.git
 ```
 
 ### With Vundle
 Add this to .vimrc:
 ```
-Bundle 'git://github.com/miripiruni/CSScomb-for-Vim.git'
+Bundle 'git://github.com/csscomb/vim-csscomb.git'
+```
+
+### With NeoBundle
+Add this to .vimrc:
+```
+NeoBundle 'csscomb/vim-csscomb', {
+        \ 'build': {
+        \     'unix': 'npm install -g csscomb',
+        \     'mac': 'npm install -g csscomb'
+        \ }}
```

Comment:
As I understand from neobundle's code, `build` is a command run once repo is cloned.
If so, line 16 is not really a requirement.
Also, here i see a list of available platforms: https://github.com/Shougo/neobundle.vim/blob/fd15e35a7a7c4e84ae62981cfa529ae7cc382330/autoload/neobundle/installer.vim#L66
We can assume there are not so many vim users on windows, but linux is worth mentioning.